MovieApp/MyCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+
+import UIKit
+import PureLayout
+
+class MyCollectionViewCell: UICollectionViewCell {
+    
+    static let cellIdentifier = "crewCellIdentifier"
+    
+    private let crewMemberNameLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+    
+    private let crewMemberPositionLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
```

Comment:
I would like you to avoid this type of view initialization and propose going with the solution that is shown in the lectures